platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceImpl.java

```diff
@@ -96,6 +97,9 @@
 
     private static final int BUNDLE_WAIT_TIMEOUT = 2;
 
+    static final EnumSet<Option> FEATURE_SERVICE_OPTIONS =
+            EnumSet.of(Option.NoAutoRefreshManagedBundles);
```

Comment:
The only change in this commit is passing the `NoAutoRefreshManagedBundles` option instead of `NoAutoRefreshBundles` when using the Karaf SeatureService to install and uninstall bundles. The rest of the changes are corresponding unit test updates and formatting.